### DISCUSSION TITLE
How to disable "x-powered-by" header ?

### DISCUSSION THREAD

**Discussion by fidgi** [loopbackio/loopback-next#8689] [Q&A] [answered]
What is the simplest way to disable the "x-powered-by" header from the underlying Express application ?

**Comment by invissiblecat** [comment]
I am using [Helmet JS ](https://helmetjs.github.io/) for this purpose. There is an example how to add it to application: https://loopback.io/doc/en/lb4/Express-middleware.html#invoke-express-middleware-explicitly-in-the-sequence

**Comment by achrinza** [comment, marked as answer]
See: https://loopback.io/doc/en/lb4/Customizing-server-configuration.html#express-settings
